### DISCUSSION TITLE
ESP8266 D1 Mini Pro (16mb)

### DISCUSSION THREAD

**Discussion by adamcoulombe** [forkineye/ESPixelStick#464] [Q&A]
Hello...

I have the 16mb D1 mini pro. (eg  [Link](https://www.aliexpress.com/item/1005001621776421.html) )

I have flashed with the latest v4.0-beta4. 

We have a breakout board we made with an SD card reader.

Is the ESP8266 D1 Mini Pro supported for SD card reading?    

When I boot up without the SD card plugged in, it boots normally. But when I put an SD card (FAT32, 32gb, blank). It will not boot, and nothing shows up in the serial monitor log.

I have tried flashing using both D1 Mini hardware setting, as well as Pixelstick V3 hardware setting.

Thank you in advance for any insight.

**Comment by garymueller** [comment]
I am having the same issue on ESP8266 and I found this [facebook post](https://www.facebook.com/groups/351083649266539/posts/574829613558607/) that may help you.  I am waiting on the resistors to test.

Solution Short answer – connect GPIO15 on ESP8266 to GND through a 10k resistor. Then connect CS from SD card directly to GPIO15.
